runtime/vm/resolvesupport.cpp

```diff
@@ -1102,6 +1122,29 @@ resolveSpecialMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
 	if (jitFlags) {
 		lookupOptions |= J9_LOOK_NO_THROW;
 	}
+
+	if (currentClass != NULL) {
+		if (resolvedClass->romClass != NULL && currentClass->romClass != NULL) {
```

Comment:
Can you clean up the formatting in here as suggested above for all the functions and then I'll take another look.